chrome/popup/affected-people.ts

```diff
@@ -0,0 +1,33 @@
+import { generateRandomNumber, sample, unique } from '../shared/util';
+
+const groups = [
+  {
+    suffix: null,
+    examples: ['MA', 'Sales', 'CX', 'Marketing', 'Finance', 'Vendor Acquisition', 'Vendor Management'],
+    prefixes: [],
+  },
+  {
+    suffix: 'vendor',
+    examples: ['Pink Stick', 'Wibble Wobble', 'Wiggle Wiggle', 'ETA', 'Booty Bump', 'Cheeri-No'],
+    prefixes: [],
+  },
+  {
+    suffix: 'customer',
+    examples: ['Loyota', 'Tradoo', 'WeWontWork', 'Runaway UK Ltd', 'Blokos Lowercase Management'],
```

Comment:
Got a few extra suggestions:
```suggestion
    examples: ['Loyota', 'Tradoo', 'WeWontWork', 'Runaway UK Ltd', 'Blokos Lowercase Management', 'BAA Print Ltd', 'Darling Bank', 'Fiber', 'Century Capital', 'UberEATS', 'Caterwings', 'Seamless', 'Feedr', 'Deliveroo', 'Feastly', 'JustStarve', 'Badon\'t', 'Loser.com', 'Seascanner', 'Tango', 'Castoff Capital', 'Seekabug', 'Spotifry', 'UWU Energy', 'Blue Steer UK', 'Sauce Dabs', 'Grumble', 'Megatough', 'List'],
```